### ISSUE TITLE
Exclude paths doesn't work in storage path with Envoyer

### ISSUE THREAD

**Issue by michellaurent** [spatie/laravel-backup#319]
I have an issue with the backup of the files in the storage_path.
I use Envoyer for deployments, so the storage_path is outside the base_path (with symlinks).

Exclude rules doesn't work. All the files are in the zip ... 
On my local computer, the exclude rules works well... 

Here's my laravel-backup config file.

```
'files' => [

    /*
     * The list of directories and files that will be included in the backup.
     */
    'include' => [
        storage_path(),
    ],

    /*
     * These directories and files will be excluded from the backup.
     *
     * Directories used by the backup process will automatically be excluded.
     */
    'exclude' => [
        storage_path('backups'),
        storage_path('app/laravel-backup'),
        storage_path('framework'),
        storage_path('logs'),
    ],

    /*
     * Determines if symlinks should be followed.
     */
    'followLinks' => true,
],
```

If I use the default config file, the exclude rule works well...

Did I miss something ?

**Comment by bbashy** [comment]
Have you tried running the command manually on the server to see if it excludes the folders?

**Comment by freekmurze** [comment]
Should be fixed in `4.3.1`
